src/styles/variables.ts

```diff
@@ -1,20 +1,5 @@
 export const variables = {
```

Comment:
Let's have the following set of variables here:
```ts
export const spacings = { ... };

export const fontFamilies = { ... };

export const typographies = { ... };

export const colors = { ... };
```

Reply:
I also created `fontSizes` because they can be used separetely from a typography style